e2e-tests/monitoring-2-0/run

```diff
@@ -27,19 +27,21 @@ kubectl_bin apply \
     -f $conf_dir/client.yml \
     -f $test_dir/conf/secrets.yml
 
-apply_cluster $test_dir/conf/$cluster.yml
+apply_cluster $test_dir/conf/$cluster-rs0.yml
 desc 'check if all 3 Pods started'
-wait_for_running $cluster 3
+wait_for_running $cluster-rs0 3
 
 # wait for prometheus
 sleep 90
 desc 'check if pmm-client container enabled'
-compare_kubectl statefulset/$cluster
-compare_kubectl service/$cluster
+compare_kubectl statefulset/$cluster-rs0
+compare_kubectl service/$cluster-rs0
+compare_kubectl statefulset/$cluster-cfg
+compare_kubectl deployment/$cluster-mongos
 
 desc 'check mongod metrics'
-get_metric_values node_boot_time_seconds $namespace-$cluster-1 admin:admin
-get_metric_values mongodb_connections $namespace-$cluster-1 admin:admin
+get_metric_values node_boot_time_seconds $namespace-$cluster-rs0-1 admin:admin
+get_metric_values mongodb_connections $namespace-$cluster-rs0-1 admin:admin
 
 #desc 'check mongos metrics'
 #get_metric_values node_boot_time ${cluster%%-rs0}-mongos-0
```

Comment:
can we uncomment mongos section and add cfg replicaset checks?

Reply:
pmm team does not fully support mongos  and as I can see not all metrics are present on pmm server (we have N/A). As for cfg I will try to do it.